benchmarks/user/advisor/run_advisor.py

```diff
@@ -68,12 +73,12 @@ def run_with_advisor(path, output, name, exec_args):
     if devito_logging is None:
         os.environ['DEVITO_LOGGING'] = 'WARNING'
 
-    with progress('Set up multi-threading environment'):
-        # Roofline analyses only make sense with threading enabled
+    with progress('Setting up multi-threading environment'):
+        # Roofline analyses are recommended with threading enabled
         os.environ['DEVITO_LANGUAGE'] = 'openmp'
 
-        # We must be able to do thread pinning, otherwise any results would be
-        # meaningless. Currently, we only support doing that via numactl
+        # Thread pinning is strongly recommended for reliable results.
+        # We support thread pinning via numactl
```

Comment:
or `KMP_AFFINITY`?

Reply:
"we support" doesn't also make much sense. We don't support anything, thread/process placement is up to the user (though we might one day devise an option to do that automatically at least for openmp since there are special for-loop clauses that allow you to specify the thread affinity for that loop. But not today)

KMP_AFFINITY is also intel stuff

The OpenMP standard env vars for thread pinning are OMP_PLACES and OMP_PROC_BIND , IIRC

Reply:
I rephrased a bit. Hope its better now